fiftyone/core/models.py

```diff
@@ -507,14 +508,15 @@ def save_batch(sample_batch, labels_batch):
                 if isinstance(imgs, Exception):
                     raise imgs
 
+                image_sizes = imgs.pop("fo_image_size", [(None, None)])
```

Comment:
`fo_image_size` is unique to `FiftyOneTransformer` so it can't be referenced in `fiftyone.core.models`.